deploy/ansible/hosts

```diff
@@ -1,8 +1,9 @@
-[centos7]
-35.187.116.62
+[hosts]
+104.155.94.58
+35.205.142.238
 
-[ubuntu16]
-35.195.150.11
 
 [all:vars]
 ansible_ssh_user=iroha
+host_key_checking=False
```

Comment:
provide comments pls

Reply:
It's a pitfall.
If I enable ssh checking (default), after recreating a remote VM (i.e google cloud) It will provide me a new VM with new keys, but same IP address, and my ssh client will reject it, as known host with changed keys.